.github/workflows/build-and-archive-docs.yml

```diff
@@ -0,0 +1,62 @@
+name: Build and archive documentation from branches
+
+on:
+  push:
+    branches:
+      - master  # Only master will build both Uyuni and SUSE Manager documentations
+      - manager-4.3 # Other branches will only build SUSE Manager documentation
+  workflow_run:
+    workflows: ["UPDATE TRANSLATION FILES"] # This workflow creates commits, so we need to run after a successful run
+    types:
+      - completed
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout this repository
+      uses: actions/checkout@v4
+      with:
+        path: uyuni-docs
+
+    - name: Checkout uyuni-docs-helper repository
+      uses: actions/checkout@v4
+      with:
+        path: uyuni-docs-helper
+        repository: uyuni-project/uyuni-docs-helper
+
+    - name: Configure environment
+      run: |
+        alias docker='podman'
+        mkdir outputs
+
+    - name: Build the Uyuni documentation
+      run: |
+        cd uyuni-docs-helper
+        ./uyuni-docs-helper -r ${{ github.ref_name }} -o /tmp -c antora-uyuni-en -p uyuni
```

Comment:
I think I am using the right target here, as this builds the doc for the site, and the PDFs, but please confirm (just in case).

Reply:
The target is correct. :+1: